### ISSUE TITLE
Server cannot restart, cannot move node

### ISSUE THREAD

**Issue by MisterQH** [zodern/meteor-docker#37]
When starting/restarting a server, we get the following error and the server is stuck in a loop.

```bash
=> Setting node version
 NODE_VERSION=14.21.4
 Node: 14.21.4 (parsed: 14.21.4)
 Using Meteor's custom NodeJS v14 LTS version
   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                  Dload  Upload   Total   Spent    Left  Speed
   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0   3 32.6M    3 1214k    0     0  7834k      0  0:00:04 --:--:--  0:00:04 7784k  42 32.6M   42 13.7M    0     0  12.3M      0  0:00:02  0:00:01  0:00:01 12.3M  80 32.6M   80 26.4M    0     0  12.1M      0  0:00:02  0:00:02 --:--:-- 12.1M  93 32.6M   93 30.6M    0     0  9524k      0  0:00:03  0:00:03 --:--:-- 9521k 100 32.6M  100 32.6M    0     0  8662k      0  0:00:03  0:00:03 --:--:-- 8660k
 mv: cannot move '/tmp/node-v14.21.4-linux-x64' to '/home/app/.nvm/versions/node/v14.21.4/node-v14.21.4-linux-x64': Directory not empty
```

We didn't had this issue a week ago.

This line seems to be the culprit:
https://github.com/zodern/meteor-docker/blob/e2ac0ff24a1886f1fe0fa3b0d41e4f1d2f004b88/image/scripts/setup_nvm.sh#L31

**Comment by MisterQH** [comment]
Related to #36

**Comment by sitiarel** [comment]
@MisterQH Did you solve the problem?

@zodern This is a critical problem, have you had a chance to look at the problem? Thanks for your attention.

**Comment by MisterQH** [comment]
@sitiarel Yes I did the fix @SeraVault suggested in https://github.com/zodern/meteor-docker/issues/36 and it seems to have fixed the issue

**Comment by zodern** [comment]
This is fixed in version 0.7.2
